src/constants/typography.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components"
+import "./typefaces.css"
+
+const fonts = {
+  workSans: `'Work Sans', sans-serif`,
```

Comment:
Generally a trend here is to name these like `body`, `header`, etc. Because if the fonts ever change the variable names no longer make sense